test/functionalTest/cases/1112_metadata_as_keymap/metadata_with_dots_v1.test

```diff
@@ -0,0 +1,179 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Metadata with dot in the name (NGSIv1)
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+# 01. Create entity with metadata with dot in the name
+# 02. Retieve entity
+# 03. Check the '=' in DB
+
+echo "01. Create entity with metadata with dot in the name"
+echo "===================================================="
+url="/v1/updateContext"
+payload='{
+    "contextElements": [
+    {
+        "type": "T",
+        "isPattern": "false",
+        "id": "E",
+        "attributes": [
+        {
+            "name": "A",
+            "type": "T",
+            "value": "foo",
+            "metadatas": [
+                {
+                    "name": "m.x",
```

Comment:
Is '=' a forbidden char in metadata name?
I imagine it is. If not we would have chosen some other char, I hope :-)